metaflow/plugins/kfp/accelerator_decorator.py

```diff
@@ -0,0 +1,40 @@
+from metaflow.decorators import StepDecorator
+
+
+class AcceleratorDecorator(StepDecorator):
+    """
+    For KFP orchestrator plugin only.
+
+    Step decorator to specify type of hardware accelerator used in a step.
+
+    To use, follow the example below.
+    ```
+    @step
+    def start(self):
+        self.ranks = list(range(self.world_size))
+        print(f"ranks: {self.ranks}")
+        self.next(self.train, foreach="ranks")
+
+    @accelerator
+    @step
+    def train(self):
+        self.rank = self.input
+        # code requiring accelerator for performance
+        ...
+    ```
+
+    Parameters
+    ----------
+    accelerator_type: str
+        Defaults to nvidia-tesla-v100.
+        More GPUs will be added based on customer needs.
+    """
+
+    name = "accelerator"
+
+    defaults = {
+        "accelerator_type": "nvidia-tesla-v100",
```

Comment:
The default should be None... nvidia-tesla-v100 is only a default for the current Zillow k8s deployment, that may change in a year.  

So better they be explicitly provided.

Reply:
`accelerator_type` is redundant and should be just `type`